### PR TITLE
Unit tests for expose_mob

### PR DESCRIPTION
**Pull request by nightred**
## About The Pull Request

This is a set of unit tests for expose_mob.
This tests that INGEST, TOUCH, VAPOR, PATCH, INJECT are triggering on standard exposure methods.

Merge after #53491 or this will fail

No CL it is a unit test

### CODE REVIEW

File: code/modules/unit_tests/reagent_mod_expose.dm [hunk 1]
```diff
@@ -0,0 +1,50 @@
+// testing the mob expose procs are working
+
+/datum/reagent/method_patch_test
+	name = "method patch test"
+
+/datum/reagent/method_patch_test/expose_mob(mob/living/target, methods=PATCH, reac_volume, show_message = TRUE)
```
Comment (Jared-Fogle) [inline code/modules/unit_tests/reagent_mod_expose.dm:6]:
```suggestion
/datum/reagent/method_patch_test/expose_mob(mob/living/target, methods = PATCH, reac_volume, show_message = TRUE)
```

File: code/modules/unit_tests/reagent_mod_expose.dm [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+/datum/unit_test/reagent_mob_expose/Run()
+	var/mob/living/carbon/human/human = allocate(/mob/living/carbon/human)
+	var/obj/item/reagent_containers/dropper/dropper = allocate(/obj/item/reagent_containers/dropper)
+	var/obj/item/reagent_containers/food/drinks/drink = allocate(/obj/item/reagent_containers/food/drinks/bottle)
+	var/obj/item/reagent_containers/pill/patch/patch = allocate(/obj/item/reagent_containers/pill/patch)
+	var/obj/item/reagent_containers/syringe/syringe = allocate(/obj/item/reagent_containers/syringe)
+
+	// INGEST
+	TEST_ASSERT_EQUAL(human.fire_stacks, 0, "Human has fire stacks on allocate")
```
Comment (Jared-Fogle) [inline code/modules/unit_tests/reagent_mod_expose.dm:21]:
```suggestion
	TEST_ASSERT_EQUAL(human.fire_stacks, 0, "Human has fire stacks before taking phlogiston")
```

**Review by Mothblocks — CHANGES_REQUESTED** [review]
My qualms are only pedantic--good work!

**Review by Mothblocks — APPROVED** [review]

File: code/modules/unit_tests/reagent_mod_expose.dm (hunk 2 above)
Comment (nightred) [reply code/modules/unit_tests/reagent_mod_expose.dm]: Cant complain with wording suggestions it always helps me

File: code/modules/unit_tests/reagent_mod_expose.dm (hunk 1 above)
Comment (nightred) [reply code/modules/unit_tests/reagent_mod_expose.dm]: done

**Pull request merged**